Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -1848,17 +1850,21 @@ namespace OpenInfraPlatform {
 				) const throw(...)
 				{
 					carve::geom::vector<3> centerToTrimPoint = trimPoint - circleCenter;
-					if (abs(centerToTrimPoint.length() - circleRadius) < 0.0001) {
+
+					if (GeomSettings()->areEqual(abs(centerToTrimPoint.length()), circleRadius)){

```

Comment:
`.length()` is always positive, no need for `abs(..)`.

Reply:
Resolved: 6e54eae